src/lib/components/ui/TextField/TextField.jsx

```diff
@@ -174,6 +190,7 @@ TextField.propTypes = {
   isLabelVisible: PropTypes.bool,
   label: PropTypes.string.isRequired,
   layout: PropTypes.oneOf(['horizontal', 'vertical']),
+  max: PropTypes.number,
```

Comment:
I don't want you to do it now, but we should start to write custom prop checks to throw error if max is set together with another than number

Reply:
I agree. I'd also like to validate mutually incompatible props in the first stage and bad design decisions in the next stage.